pkg/gds/emails/client.go

```diff
@@ -298,9 +311,16 @@ func (m *EmailManager) SendDeliverCertificates(vasp *pb.VASP, path string) (sent
 				continue
 			}
 
+			sent++
+
+			if err = models.AppendEmailLog(contact, string(admin.ResendDeliverCerts), msg.Subject); err != nil {
+				log.Error().Err(err).Str("vasp", vasp.Id).Int("contact", idx).Msg("could not log deliver certs email")
+				nErrors++
+				continue
+			}
```

Comment:
We're modifying the VASP record here, I'm assuming that at some point later in the function call the VASP is saved back to disk with the changes to its contacts?